### PR TITLE
knitcitations

### PR DESCRIPTION
**Pull request by vivekbhr**
adds the knitcitations package

### CODE REVIEW

**Comment by dpryan79** [comment]
If you end up needing a hand with this let me know, I've done quite a few of these recently.

**Comment by vivekbhr** [comment]
@dpryan79 if you could help with the failing build with appveyor. IDK why it can't find the library after the installation seems to be ok

File: recipes/r-knitcitations/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/knitcitations_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/knitcitations/knitcitations_{{ version }}.tar.gz
+  sha256: 3b366d6b7cd02998d451f820a7883e6c5c315391a5301112a80c8b725969081e
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+
```
Comment (cbrueffer) [inline recipes/r-knitcitations/meta.yaml]: Not sure why the win build is failing, but I've seen similar problems with a few packages. You can disable it for now and we'll figure it out later.

File: recipes/r-knitcitations/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    is pulled from the web rather than having to be entered into a local database such
+    as 'bibtex' ahead of time. The package is primarily aimed at authoring in the R
+    'markdown' format, and can provide outputs for web-based authoring such as linked
+    text for inline citations.  Cite using a 'DOI', URL, or 'bibtex' file key.  See
+    the package URL for details.
+  license_family: MIT
+
+extra:
+   recipe-maintainers:
+     - vivekbhr
```
Comment (cbrueffer) [inline recipes/r-knitcitations/meta.yaml]: Could you add a few more maintainers?

    - johanneskoester
    - bgruening
    - daler
    - jdblischak
    - cbrueffer
    - dbast

File: recipes/r-knitcitations/meta.yaml [hunk 3]
```diff
@@ -14,7 +14,7 @@ source:
   sha256: 3b366d6b7cd02998d451f820a7883e6c5c315391a5301112a80c8b725969081e
 
 build:
-  merge_build_host: True  # [win]
+  merge_build_host: False  # [win]
```
Comment (cbrueffer) [inline recipes/r-knitcitations/meta.yaml]: This line should stay as it is, but you can add a line "skip: true  # [win]"

Comment (vivekbhr) [reply recipes/r-knitcitations/meta.yaml]: thanks for the tips. I fixed it now :+1:

**Pull request merged**